2023-03-09 19:27:42,888 INFO    MainThread:4780 [wandb_setup.py:_flush():68] Configure stats pid to 4780
2023-03-09 19:27:42,888 INFO    MainThread:4780 [wandb_setup.py:_flush():68] Loading settings from /Users/christoszacharopoulos/.config/wandb/settings
2023-03-09 19:27:42,888 INFO    MainThread:4780 [wandb_setup.py:_flush():68] Loading settings from /Users/christoszacharopoulos/projects/dynamic_risk/src/wandb/settings
2023-03-09 19:27:42,888 INFO    MainThread:4780 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2023-03-09 19:27:42,888 INFO    MainThread:4780 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program_relpath': 'src/ingestion_01.py', 'program': '/Users/christoszacharopoulos/projects/dynamic_risk/src/ingestion_01.py'}
2023-03-09 19:27:42,888 INFO    MainThread:4780 [wandb_init.py:_log_setup():476] Logging user logs to /Users/christoszacharopoulos/projects/dynamic_risk/src/wandb/run-20230309_192742-38tiamo5/logs/debug.log
2023-03-09 19:27:42,888 INFO    MainThread:4780 [wandb_init.py:_log_setup():477] Logging internal logs to /Users/christoszacharopoulos/projects/dynamic_risk/src/wandb/run-20230309_192742-38tiamo5/logs/debug-internal.log
2023-03-09 19:27:42,888 INFO    MainThread:4780 [wandb_init.py:_jupyter_setup():426] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x149608ac0>
2023-03-09 19:27:42,889 INFO    MainThread:4780 [wandb_init.py:init():516] calling init triggers
2023-03-09 19:27:42,889 INFO    MainThread:4780 [wandb_init.py:init():519] wandb.init called with sweep_config: {}
config: {}
2023-03-09 19:27:42,889 INFO    MainThread:4780 [wandb_init.py:init():569] starting backend
2023-03-09 19:27:42,889 INFO    MainThread:4780 [wandb_init.py:init():573] setting up manager
2023-03-09 19:27:42,891 INFO    MainThread:4780 [backend.py:_multiprocessing_setup():102] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2023-03-09 19:27:42,893 INFO    MainThread:4780 [wandb_init.py:init():580] backend started and connected
2023-03-09 19:27:42,897 INFO    MainThread:4780 [wandb_run.py:_label_probe_notebook():1113] probe notebook
2023-03-09 19:27:42,897 INFO    MainThread:4780 [wandb_run.py:_label_probe_notebook():1123] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-03-09 19:27:42,897 INFO    MainThread:4780 [wandb_init.py:init():658] updated telemetry
2023-03-09 19:27:42,922 INFO    MainThread:4780 [wandb_init.py:init():693] communicating run to backend with 60 second timeout
2023-03-09 19:27:43,810 INFO    MainThread:4780 [wandb_run.py:_on_init():2000] communicating current version
2023-03-09 19:27:44,040 INFO    MainThread:4780 [wandb_run.py:_on_init():2004] got version response upgrade_message: "wandb version 0.13.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-03-09 19:27:44,040 INFO    MainThread:4780 [wandb_init.py:init():728] starting run threads in backend
2023-03-09 19:27:45,186 INFO    MainThread:4780 [wandb_run.py:_console_start():1980] atexit reg
2023-03-09 19:27:45,186 INFO    MainThread:4780 [wandb_run.py:_redirect():1838] redirect: SettingsConsole.WRAP_RAW
2023-03-09 19:27:45,186 INFO    MainThread:4780 [wandb_run.py:_redirect():1903] Wrapping output streams.
2023-03-09 19:27:45,186 INFO    MainThread:4780 [wandb_run.py:_redirect():1925] Redirects installed.
2023-03-09 19:27:45,186 INFO    MainThread:4780 [wandb_init.py:init():765] run started, returning control to user process
2023-03-09 19:27:45,354 INFO    MainThread:4780 [wandb_run.py:_finish():1746] finishing run christonikos/dynamic_risk_assesment/38tiamo5
2023-03-09 19:27:45,354 ERROR   MainThread:4780 [jupyter.py:save_history():440] Run pip install nbformat to save notebook history
2023-03-09 19:27:45,354 INFO    MainThread:4780 [jupyter.py:save_ipynb():378] not saving jupyter notebook
2023-03-09 19:27:45,354 INFO    MainThread:4780 [wandb_init.py:_jupyter_teardown():408] cleaning up jupyter logic
2023-03-09 19:27:45,354 INFO    MainThread:4780 [wandb_run.py:_atexit_cleanup():1949] got exitcode: 0
2023-03-09 19:27:45,355 INFO    MainThread:4780 [wandb_run.py:_restore():1932] restore
2023-03-09 19:27:45,355 INFO    MainThread:4780 [wandb_run.py:_restore():1938] restore done
2023-03-09 19:27:50,134 INFO    MainThread:4780 [wandb_run.py:_footer_history_summary_info():3377] rendering history
2023-03-09 19:27:50,134 INFO    MainThread:4780 [wandb_run.py:_footer_history_summary_info():3409] rendering summary
2023-03-09 19:27:50,135 INFO    MainThread:4780 [wandb_run.py:_footer_sync_info():3333] logging synced files
